mobile/native/src/api.rs

```diff
@@ -177,21 +177,28 @@ pub fn subscribe(stream: StreamSink<event::api::Event>) {
 /// There is no recovery possible, and panicking is the only option ensuring
 /// that we will get an adequate crash report.
 pub fn run_in_flutter(config: Config, app_dir: String, seed_dir: String) {
-    run(config, app_dir, seed_dir).expect("Failed to start the backend");
-}
-
-pub fn run(config: Config, app_dir: String, seed_dir: String) -> Result<()> {
-    std::panic::set_hook(
-        #[allow(clippy::print_stderr)]
-        Box::new(|info| {
-            let backtrace = Backtrace::force_capture();
-
-            tracing::error!(%info, "Aborting after panic in task");
-            eprintln!("{backtrace}");
-
-            std::process::abort()
-        }),
-    );
+    run(config, app_dir, seed_dir, true).expect("Failed to start the backend");
+}
+
+pub fn run(
+    config: Config,
+    app_dir: String,
+    seed_dir: String,
+    backtrace_on_panic: bool,
```

Comment:
😛 Surely you would advocate for an `enum` here.

Reply:
enum it is then! :)